packages/hierarchies/src/hierarchies/internal/operators/Grouping.ts

```diff
@@ -88,34 +100,33 @@ async function groupInstanceNodes(
   nodes: ProcessedInstanceHierarchyNode[],
   extraSiblings: number,
   groupingHandlers: GroupingHandler[],
-  onGroupingNodeCreated: undefined | ((groupingNode: ProcessedGroupingHierarchyNode) => void),
+  parentNode: ParentHierarchyNode | undefined,
```

Comment:
```suggestion
  parentNode?: ParentHierarchyNode,
```

Reply:
In this context parent node is not optional - `undefined` means we're requesting root nodes. So I'd prefer to leave it as is.